tests/sentry/api/endpoints/test_organization_metric_data.py

```diff
@@ -1782,77 +1768,33 @@ def test_incorrect_crash_free_rate(self):
-        self._send_buckets(
-            [
-                {
-                    "org_id": org_id,
-                    "project_id": self.project.id,
-                    "metric_id": self.session_metric,
-                    "timestamp": (user_ts // 60 - 4) * 60,
-                    "tags": {
-                        self.session_status_tag: rh_indexer_record(org_id, "errored_preaggr"),
-                    },
-                    "type": "c",
-                    "value": 10,
-                    "retention_days": 90,
-                },
-                {
-                    "org_id": org_id,
-                    "project_id": self.project.id,
-                    "metric_id": self.session_metric,
-                    "timestamp": (user_ts // 60 - 4) * 60,
-                    "tags": {
-                        self.session_status_tag: rh_indexer_record(org_id, "crashed"),
-                    },
-                    "type": "c",
-                    "value": 2,
-                    "retention_days": 90,
-                },
-                {
-                    "org_id": org_id,
-                    "project_id": self.project.id,
-                    "metric_id": self.session_metric,
-                    "timestamp": (user_ts // 60 - 4) * 60,
-                    "tags": {
-                        self.session_status_tag: rh_indexer_record(org_id, "abnormal"),
-                    },
-                    "type": "c",
-                    "value": 4,
-                    "retention_days": 90,
-                },
-                {
-                    "org_id": org_id,
-                    "project_id": self.project.id,
-                    "metric_id": self.session_metric,
-                    "timestamp": user_ts,
-                    "tags": {
-                        self.session_status_tag: rh_indexer_record(org_id, "init"),
-                    },
-                    "type": "c",
-                    "value": 15,
-                    "retention_days": 90,
-                },
-            ],
-            entity="metrics_counters",
-        )
-        self._send_buckets(
-            [
-                {
-                    "org_id": org_id,
-                    "project_id": self.project.id,
-                    "metric_id": self.session_error_metric,
-                    "timestamp": user_ts,
-                    "tags": {tag: value},
-                    "type": "s",
-                    "value": numbers,
-                    "retention_days": 90,
-                }
-                for tag, value, numbers in (
-                    (self.release_tag, rh_indexer_record(org_id, "foo"), list(range(3))),
-                )
-            ],
-            entity="metrics_sets",
-        )
+        for tag, tag_value, value in (
+            ("session.status", "errored_preaggr", 10),
+            ("session.status", "crashed", 2),
+            ("session.status", "abnormal", 4),
+            ("session.status", "init", 15),
+        ):
```

Comment:
Can we simplify this just by passing the `tag` as always `session.status` since it does not change